src/components/QuranReader/selectVerseHighlight.ts

```diff
@@ -0,0 +1,46 @@
+import { selectAudioFile, selectCurrentTime } from 'src/redux/slices/AudioPlayer/state';
+import Verse from 'types/Verse';
+import inRange from 'lodash/inRange';
+
```

Comment:
// Returns whether the verse should be highlighted given the current audio timestamp